tensorflow/core/common_runtime/bfc_allocator.cc

```diff
@@ -282,7 +283,7 @@ bool BFCAllocator::DeallocateFreeRegions(size_t rounded_bytes)
   for (const AllocationRegion& region : region_manager_.regions()) {
     ChunkHandle h = region_manager_.get_handle(region.ptr());
     bool any_use = false;
-    while (h != kInvalidChunkHandle) {
+    while (static_cast<int>(h) != kInvalidChunkHandle) {
```

Comment:
Changing the type of `kInvalidChunkHandle` should allow us to get rid of these casts, right?